experimenter/experimenter/features/tests/fixtures/valid_features/firefox-desktop/experimenter.yaml

```diff
@@ -47,3 +47,17 @@ prefSettingFeature:
       type: boolean
       setPref: nimbus.test.boolean
       description: Boolean property
+newSetPrefFeature:
```

Comment:
Is it at all feasible to change the other one to `oldSetPrefFeature`? :|

Reply:
That is probably a better idea, I was just being lazy :)